### ISSUE TITLE
Windows 10 x64 - Network traffic does not register

### ISSUE THREAD

**Issue by jsuelwald** [GitSquared/edex-ui#256]

**Comment by GitSquared** [comment]
Would you mind providing a bit more details?

**Comment by waqasrana** [comment]
I'm on mac, and i dont see any traffic in network either. Attaching screenshot, let me know how can i provide more info

![screen shot 2018-11-23 at 2 32 33 pm](https://user-images.githubusercontent.com/16499724/48936428-bd665580-ef2c-11e8-9b06-a827429e48ff.png)

**Comment by waqasrana** [comment]
port useage:

```
eDEX-UI   23136          <user>   46u  IPv6 0xa146aeb3f13e5ae3      0t0    TCP *:3000 (LISTEN)
eDEX-UI   23136          <user>   60u  IPv4 0xa146aeb41414495b      0t0    TCP 127.0.0.1:51264->127.0.0.1:3000 (ESTABLISHED)
eDEX-UI   23136          <user>   61u  IPv6 0xa146aeb3f13e6023      0t0    TCP 127.0.0.1:3000->127.0.0.1:51264 (ESTABLISHED)
eDEX-UI   23139          <user>   29u  IPv4 0xa146aeb4141cc253      0t0    TCP 192.168.80.27:51623->104.18.127.144:443 (ESTABLISHED)
```

**Comment by GitSquared** [comment]
@waqasrana Do any errors pop up when starting edex from the command line? Do you have a network proxy or something that might block it from accessing the internet?

**Comment by waqasrana** [comment]
No proxy. in attached screenshot app could ping 8.8.8.8 using app. Its added to allowed list of my firewall as well (just to be sure i relaunched it with firewall disabled as well).

running from terminal gives following:

```
eDEX-UI.app/Contents/MacOS/eDEX-UI
▶  start     Starting eDEX-UI v1.0.1
ℹ  info      With Node 8.9.3 and Electron 2.0.13
ℹ  info      Renderer is Chrome 61.0.3163.100
▶  Startup   Initialized timer...
ℹ  info      Base config dir is /Users/Waqas/Library/Application Support/eDEX-UI
☐  pending   Loading settings file...
✔  success   Settings loaded!
☐  pending   Creating new terminal process on port 3000
✔  success   Terminal back-end initialized!
ℹ  info      Creating window...
…  watching  Waiting for frontend connection...
✔  success   Connected to frontend!
◼  Startup   Timer run for: 12.22s
ℹ  info      Resized TTY to  077 037
ℹ  info      Resized TTY to  120 035
ℹ  info      UpdateChecker: Running latest version.
```
the "boot" screen shows no error either.

**Comment by waqasrana** [comment]
also my correct external ip blinks in ip section (keeps switching between dashs and ip every second), while rest stays same (offline, no ms, no traffic)

**Comment by lt292** [comment]
same issues.  i am in China, and the google site's is not reachable, is this the reason?
my OS is windows 10 1809 64bit

**Comment by GitSquared** [comment]
@lt292 Might be the reason indeed. There's a new setting coming in v1.2 that will allow you to change the IP used for testing connectivity (currently it's `1.1.1.1`, Cloudflare's public DNS).

**Comment by MrTucks** [comment]
I have a similar issue. I see my IP adress and ping but the graph shows 0 mb up and down. I think this might be because I have 2 Ethernet adapters. Actually one real network adapter and a second one because of VirtualBox virtual machines. In the windows task manager the VirtualBox network adapter is the first one in the list. Maybe eDex tries to show traffic from that one instead of the real actual network adapter.

**Comment by GitSquared** [comment]
This was fixed in ef82aed; the conninfo module now uses the network interface selected by the netstat module (the one that shows ping & ip address).

Also the IP address used to test latency (ping) is configurable since v1.2 release.
